README.md

```diff
@@ -779,11 +779,11 @@ other `.swiftlint.yml` files somewhere within the directory. **So if you want to
 
 ## License
 
-[MIT licensed.](LICENSE)
+[MIT licensed.](https://raw.githubusercontent.com/realm/SwiftLint/main/LICENSE)
```

Comment:
I've made the link ["https://raw.githubusercontent.com/realm/SwiftLint/main/LICENSE"](https://raw.githubusercontent.com/realm/SwiftLint/main/LICENSE), but I wonder if [this one](https://github.com/realm/SwiftLint/blob/main/LICENSE), which takes you directly to GitHub, would be better?🤔

Reply:
Oh sorry, I missed your comment ...

Yes, that link would indeed be better, because it also offers GitHub's license details.

Reply:
Created #5127 to fix the links. Thank you for the hint!